app/views/admin/site_customization/pages/_form.html.erb

```diff
@@ -30,33 +31,20 @@
       <br>
     <% end %>
   </div>
-  <div class="small-12 medium-4 column">
-    <%= f.label :locale %>
-    <%= f.select :locale, I18n.available_locales.map { |l| [name_for_locale(l), l.to_s] }, { include_blank: true, label: false } %>
-  </div>
+
   <div class="small-12 medium-4 column">
     <%= f.label :slug %>
     <%= f.text_field :slug, label: false, size: 80, maxlength: 80 %>
   </div>
-
-    <div class="small-12 column">
-      <hr>
-
-      <%= f.label :title %>
-      <%= f.text_field :title, label: false %>
-
-      <%= f.label :subtitle %>
-      <%= f.text_field :subtitle, label: false, size: 80, maxlength: 80 %>
-
-      <div class="ckeditor margin-top">
-        <%= f.label :content %>
-        <%= f.cktext_area :content, label: false, cols: 80, rows: 10,
-                          ckeditor: { language: I18n.locale, toolbar: "admin" } %>
-      </div>
-
-      <div class="small-12 medium-6 large-3">
-        <%= f.submit class: "button success expanded" %>
-      </div>
+  <div class="small-12 column">
+	<hr>  
```

Comment:
Could you remove the trailing whitespaces in this line? By the way, IMHO this line should be either indented or before the previous line (I'm not sure which one is right; you probably know it better).